src/main/java/at/ac/tuwien/kr/alpha/common/predicates/ExternalNativeBiPredicate.java

```diff
@@ -0,0 +1,23 @@
+package at.ac.tuwien.kr.alpha.common.predicates;
+
+import at.ac.tuwien.kr.alpha.common.terms.ConstantTerm;
+
+import java.util.List;
+
+public class ExternalNativeBiPredicate<T, U> extends BaseExternalNativePredicate {
```

Comment:
Do we have non-`Native` predicates? I suggest simply dropping `Native` in the name.

Reply:
Done.